src/rules/eol_last.coffee

```diff
@@ -10,4 +10,11 @@ module.exports = class EOLLast
     lintLine: (line, lineApi) ->
         return null unless lineApi.isLastLine()
 
-        return true if line.length
+        isNewline = line.length is 0
+
+        previousIsNewline = if lineApi.lineCount > 1
+            lineApi.lines[lineApi.lineNumber - 1].length is 0
+        else
+            no
```

Comment:
can you make this false. i dont think we use no at all anywhere in the coffeelint code.